modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -71,6 +71,12 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 {productname} 6.5 also includes the following bug fixes:
 
+=== Removing an image that failed to upload from an empty paragraph would leave the paragraph without a padding br.
+//#TINY-9696
+
+In previous versions of {productname}, images failed to upload within an empty paragraph, the absence of a successful upload would cause the paragraph to lack the expected padding. This occurs when the failed upload is removed, leaving the paragraph without the necessary spacing.
+
+Consequently, the affected paragraph may have appeared visually inconsistent or improperly formatted. To address this issue, it is recommended to add the appropriate padding to ensure the paragraph is visually balanced and displays as intended. By adding the required padding, the paragraph will regain its intended structure and avoid any unwanted visual disruptions.
```

Comment:
```suggestion
=== Removing an image that failed to upload from an empty paragraph would leave the paragraph without a padding `<br />` tag.
//#TINY-9696

Previously, when an image upload into an otherwise empty paragraph failed, the upload failure resulted in the empty paragraph not having the expected `<br />` tag to serve as padding.

When the upload failure was noted by a {productname} editor instance, and the placeholder material for the expected image was removed by the editor, the expected padding was not set.

The empty paragraph element was, in this circumstance, rendered inaccessible. And it presented to the end-user as the empty paragraph element being removed or improperly formatted.

In {productname} 6.5, when the `images_uploade_handler` gets a `remove: true` callback, the parent element is checked and, if required padding is not present, it is added.
```

Edits in line with @HAFRMO’s [notes](https://github.com/tinymce/tinymce-docs/pull/2698/files#r1230889142).